src/sass/base/_b-utils.scss

```diff
@@ -5,3 +5,43 @@
 .no-background-image {
   background-image: none;
 }
+
+.va-button-link {
```

Comment:
Yeah, anchors without `href` attributes can't be tabbed to by default, plus `a` tags are meant for links, not actions.